.github/workflows/pull-request-checks.yml

```diff
@@ -42,12 +41,4 @@ jobs:
         npm run build
         npm run test
       env:
-        CI: false # TODO: revisit this. npm build turns extra checks on when this is true and something there is broken
-    - uses: jacobsee/s2i-github-action@master
-      with:
-        base: registry.access.redhat.com/rhscl/httpd-24-rhel7
-        path: build
-        output_image: quay.io/rht-labs/omp-frontend:latest # TODO: Tag using something better. _Very_ high priority before release.
-        image_push_registry: quay.io
-        image_push_username: ${{ secrets.QUAY_USERNAME }}
-        image_push_password: ${{ secrets.QUAY_PASSWORD }}
+        CI: false # TODO: revisit this. npm build turns extra checks on when this is true and something there is broken
```

Comment:
I'm fine with going ahead and setting this to true. I can deal with the fallout and fix any issues. I'm working on the linting rules rn, so that will probably line up.

Reply:
Okay, if you're good with that, I'll add it!